src/pages/VestingClaim.tsx

```diff
@@ -119,84 +118,106 @@ function VestingClaim(): ReactElement {
   }
 
   return (
-    <>
-      <div className={styles.container}>
-        <div className={styles.gradient}></div>
-        <div className={styles.logoWrapper}>
-          <div className={styles.logo}>
-            <img src={logo} />
-          </div>
-        </div>
-        <div className={styles.mainContent}>
-          {isValidBeneficiary ? (
-            <>
-              <div className={styles.amountContainer}>
-                <div>
-                  <div>
-                    {t("investorClaimAmount", {
-                      claimable: formatAmount(claimableVestedAmount),
-                    })}
-                  </div>
-                  <div>
-                    {t("investorRemainingAmount", {
-                      remaining: formatAmount(remainingAmount),
-                    })}
-                  </div>
-                </div>
-                {canAdd && (
-                  <img
-                    src={plusIcon}
-                    className={styles.plus}
-                    style={{ width: 16, marginBottom: 3, marginLeft: 32 }}
-                    onClick={() => addToken()}
-                  />
-                )}
+    <div>
+      <Box
+        py={5}
+        sx={{
+          backgroundImage: (theme) =>
+            theme.palette.mode === "dark"
+              ? "linear-gradient(180deg, rgba(0,0,0,0) 0%, rgba(7,7,19,1) 25%, rgba(18,19,52,1) 50%,rgba(0,0,0,0) 50%, rgba(0,0,0,0) 100%)"
+              : `linear-gradient(180deg, #FFF 25%, #FAF3CE 50%,#FFF 50%, #FFF 100%)`,
+        }}
+      >
+        <Box
+          display="flex"
+          justifyContent="center"
+          alignItems="center"
+          width={170}
+          height={170}
+          borderRadius="50%"
+          marginX="auto"
+          boxShadow="0px 4px 20px rgba(255, 255, 255, 0.25)"
+          sx={{
+            background: (theme) =>
+              theme.palette.mode === "dark"
+                ? "linear-gradient(0deg, #000 0%, #341291 100%)"
+                : "linear-gradient(180deg, #FFF 0%, #FBF4CF 100%)",
+          }}
+        >
+          <img src={logo} width={138} height={138} />
+        </Box>
+      </Box>
+
+      <Box textAlign="center">
+        {isValidBeneficiary ? (
+          <>
+            <Box display="flex" justifyContent="center" alignItems="center">
+              <div>
```

Comment:
Because I have never used this page before, but I think `Claimable `and `Total SDL` should be on different line.
![image](https://user-images.githubusercontent.com/75422800/164507319-3f09726e-a57b-449d-b79b-d9cd8e465a72.png)
But if we use fragments, then  `Claimable `and `Total SDL` could be inline.
![image](https://user-images.githubusercontent.com/75422800/164507529-91cb3984-38e4-49aa-abda-8415a34ad70b.png)

So, I believe we need `div` in here.